### PR TITLE
Adds documentation for analytics dashboards customization.

### PR DESCRIPTION
**Pull request by Ketharan**
## Purpose
This PR adds documentation for customization of dashboards in APIM analytics. This content should be available in 3.1.0 doc space.

### CODE REVIEW

File: en/docs/learn/analytics/customizing-analytics-dashboards.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+    
+3.  A form will be displayed as shown below. Add a valid `Name` and `URL` for the dashboard according to the preference and click the `ok` button.
+
+    ![Dashboard duplication form](../../assets/img/learn/apim-analytics-dashboard-duplication-form.png)
+    
+4.  Now a copy of the dashboard is created with the provided `Name` as shown below.
+    
+    ![Duplicated dashboard](../../assets/img/learn/apim-analytics-duplicated-dashboard.png)
+    
+5.  Again click the dropdown button at the bottom right corner of the created dashboard and click the `design` button as shown below.
```
Comment (fazlan-nazeem) [inline en/docs/learn/analytics/customizing-analytics-dashboards.md]: let's remove the word 'Again'

File: en/docs/learn/analytics/customizing-analytics-dashboards.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
+#Customizing Analytics Dashboards
+
+API Manager analytics provides three types of dashboards which are APIM Admin, APIM Developer Portal and APIM Publisher.
+
+The users will not be allowed to modify the default dashboards, i.e either to modify the layout of the widget or add custom widgets to a particular dashboard. If it is required to modify the default dashboard, a copy of the dashboard should be created and the modification should be applied to that copy as described below.
```
Comment (fazlan-nazeem) [inline en/docs/learn/analytics/customizing-analytics-dashboards.md]: "If it is required to modify one of the default dashboards"

Comment (Ketharan) [reply en/docs/learn/analytics/customizing-analytics-dashboards.md]: Resolved with e0624b9b657a46ee271b237d2dcc9e7fb652a7b1

File: en/docs/learn/analytics/customizing-analytics-dashboards.md (hunk 1 above)
Comment (Ketharan) [reply en/docs/learn/analytics/customizing-analytics-dashboards.md]: Resolved with ba9b2a4d35f938794a98f8eb2d67f37c0f9f911a

**Pull request merged**